pulumi/infra/graphql.py

```diff
@@ -55,6 +55,7 @@ def __init__(
                     "UX_BUCKET_URL": pulumi.Output.concat(
                         "https://", ux_bucket.bucket_regional_domain_name
                     ),
+                    "GRAPL_SCHEMA_PROPERTIES_TABLE": db.schema_properties_table,
```

Comment:
this appears to be failing the typecheck, is there a `.name` property we can dereference here?